### ISSUE TITLE
Remove inih subproject

### ISSUE THREAD

**Issue by stephanlachnit** [FeralInteractive/gamemode#195]
Currently working on getting gamemode 1.5 packed for Debian, and we try to put our build files just on top of this repo, as it is good practice to keep track of all changes. However the inih submodule is really a struggle for external build systems, and also not really that nice considering how old the submodule state in this repo really is (2 years old). Since inih is packed for Debian and Fedora ([pkgs.org](https://pkgs.org/download/inih)), I don't see any good reason to keep this.
Could you do this? I could even write the PR if you don't have the time for it.

**Comment by mdiluz** [comment]
Can you describe the issues you're facing in more detail? I can fix this up but want to make sure the fix actually helps

**Comment by stephanlachnit** [comment]
Right now the package is in a non-forked repo, meaning all changes to code from gamemode need to be uploaded by hand. I worked on putting all build files in a separate branch, but here is a problem. When gamemode is built, meson downloads inih, which will cause conflicts with the upstream tarball (inih is not included in the github tarball), so that's one problem for Debian, but there is a workaround for that (ignore the directory). More problematic is that the build environment ignores the .git folder, since it might be used for the packaging itself. When you now build in a clean chroot with just the source code and not the .git folder, the build will fail from forked git repo since the inih submodules won't be downloaded.

Tbh one could say that this is the problem of Debian, and not from gamemode, but I feel there should be a better solution, considering that the submodule state is quite old. As I said, on Debian (libinih-dev) and Fedora (inih-devel) systems, there are packages which include `/usr/include/ini.h`, as well a `libinih.so` in the lib folder.

I tried to write a patch for the Debian package to ignore the submodule and use Debian's inih package, but I don't know how to link against it. Probably due to my knowledge of meson, sadly inih is not in pkg-config (I will try to change that).
